src/nni_manager/training_service/common/remoteMachineData.ts

```diff
@@ -7,7 +7,8 @@ import * as fs from 'fs';
 import { Client, ConnectConfig } from 'ssh2';
```

Comment:
this file is only used by remote node, suggest to keep it under /remote_machine/

Reply:
moved to remote_machine